app/helpers/information_required_helper.rb

```diff
@@ -0,0 +1,5 @@
+module InformationRequiredHelper
```

Comment:
As this will only be used for the email, maybe this should be called MailerHelper?